# Header
std_msgs/Header header
# Features
serpent/StereoFeature[] features
# Left image camera info (optional)
sensor_msgs/CameraInfo left_info
# Right image camera info (optional)
sensor_msgs/CameraInfo right_info
